Frontend/package.json

```diff
@@ -61,7 +60,7 @@
     "typescript": "^3.8"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "export REACT_APP_BACKEND_URL=https://localhost:44301 && react-scripts start",
```

Comment:
This would make things a little difficult to run on windows. Best run the start command as:

REACT_APP_BACKEND_URL=https://localhost:44301 npm run start

instead.

Reply:
Removed the env export